vllm_ascend/quantization/w8a8_dynamic.py

```diff
@@ -152,7 +157,11 @@ def fused_experts_with_mc2(
     if quant_mode == 0:
         dynamic_scale = None
 
-    down_out_list = apply_mlp(expand_x,
+    # place hidden_states in a list to transfer its ownership into the `apply_mlp` function
+    hidden_states_wrapper = [expand_x]
+    del expand_x
```

Comment:
There still a lot of del action everywhere. We can create a new func like `release_tenstor` to make the `del` aciton more clear for developer